playground/package.json

```diff
@@ -1,43 +1,47 @@
 {
-  "name": "prql-playground",
-  "version": "0.1.0",
-  "private": true,
+  "browserslist": {
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ],
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ]
+  },
   "dependencies": {
     "@monaco-editor/react": "^4.4.5",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^14.2.0",
     "monaco-editor": "^0.33.0",
-    "prql-js": "^0.1.3",
+    "prql-js": "file:../prql-js/pkg",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "wasm-react-scripts": "5.0.2",
     "react-syntax-highlighter": "^15.5.0",
+    "wasm-react-scripts": "5.0.2",
     "web-vitals": "^2.1.4"
   },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+  "devDependencies": {
+    "npm-watch": "^0.11.0"
   },
   "eslintConfig": {
     "extends": [
       "react-app",
       "react-app/jest"
     ]
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "homepage": "https://prql-lang.org/playground/",
+  "name": "prql-playground",
+  "private": true,
+  "scripts": {
+    "build": "react-scripts build",
+    "eject": "react-scripts eject",
+    "preinstall": "cd ../prql-js && wasm-pack build",
```

Comment:
This file is mostly autoformatting changes annoyingly, but this is the important line